src/conda/devcontainer-feature.json

```diff
@@ -1,7 +1,8 @@
 {
     "id": "conda",
     "version": "1.0.2",
```

Comment:
version bump? 😄

Reply:
I believe that's still needed to reflect this change in OCI for UX changes? or no?

Reply:
yes good point!

Reply:
Although...not strictly needed in this case since the index file will end up being updated after concatenating this new config.  But nonetheless good practice.